CLAUDE.md

```
@@ -143,4 +144,36 @@
 ### 汎用コンポーネント
 - Selectボックス：@yps-crispy-carnival/src/components/ui/Select/index.tsx  
 - Formのカード：@yps-crispy-carnival/src/components/ui/FormCard/index.tsx
-- ページタイトル：@yps-crispy-carnival/src/components/ui/Title/index.tsx
\ No newline at end of file
+- ページタイトル：@yps-crispy-carnival/src/components/ui/Title/index.tsx
+- モーダルダイアログ：@yps-crispy-carnival/src/components/ui/Dialog/index.tsx
+   - ビジネスロジック側で利用する場合、○○Modal/でディレクトリを切り、index.tsx, index.stories.tsxを切り出すこと
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)